cmd2/cmd2.py

```diff
@@ -1874,22 +1874,26 @@ def _redirect_output(self, statement: Statement) -> Tuple[bool, utils.Redirectio
                 self.perror("Cannot redirect to paste buffer; missing 'pyperclip' and/or pyperclip dependencies")
                 redir_error = True
 
+            # Redirecting to a file
             elif statement.output_to:
-                # going to a file
-                mode = 'w'
-                # statement.output can only contain
-                # REDIRECTION_APPEND or REDIRECTION_OUTPUT
+                # statement.output can only contain REDIRECTION_APPEND or REDIRECTION_OUTPUT
                 if statement.output == constants.REDIRECTION_APPEND:
                     mode = 'a'
+                else:
+                    mode = 'w'
+                mode += 't'
```

Comment:
I thought text mode is the default and thus that this is redundant but explicit?

Reply:
You are correct, it is redundant.  I removed it.